frames/window_breakdown/window_playerbreakdown_compare.lua

```diff
@@ -819,19 +819,30 @@ local on_enter_target = function(self)
                     bar [2].lefttext:SetText(this_index .. ". " .. name)
                     bar [2].bg:Show()
 
+
                     if (spell [2] > this_spell [2]) then
                         local diff = spell [2] - this_spell [2]
-                        local up = diff / this_spell [2] * 100
-                        up = math.floor(up)
+                        local up
+                        if (this_spell [2] == 0) then
+                            up = 999
+                        else
+                            up = diff / this_spell [2] * 100  --Div by 0
+                            up = math.floor(up)
+                        end
                         if (up > 999) then
                             up = "" .. 999
                         end
                         bar [2].righttext2:SetText(Details:ToK2Min (this_spell [2]))
                         bar [2].righttext:SetText(" |c" .. minor .. up .. "%|r")
                     else
                         local diff = this_spell [2] - spell [2]
-                        local down = diff / spell [2] * 100
-                        down = math.floor(down)
+                        local down
+                        if (spell [2] == 0) then
+                            down = 999
+                        else
+                            down = diff / spell [2] * 100  -- Div by 0 
+                            down = math.floor(down)
+                        end
```

Comment:
Use `math.max` instead of this weird block of code.

`math.max(spell[2], 0.01)` will avoid divide by zeros while still keeping it valid. This is what we tend to do with potential divide by zeros in general.